README.md

```diff
@@ -335,6 +335,12 @@ Information 15.8 (2024): 472.
 IEEE Sensors Reviews (2024), vol 1, pp. 14-63. [PDF](https://ieeexplore.ieee.org/iel8/10347229/10787061/10795229.pdf)
 
 
+- <a name="Iddrisu2025EventEyeSurvey"></a>Iddrisu, K., Shariff, W., Corcoran, P., O’Connor, N., Lemley, J., Little, S.,  
+*[Event Camera Based Eye Motion Analysis: A Survey]([https://ieeexplore.ieee.org/document/XXXXXXX](https://ieeexplore.ieee.org/abstract/document/10681084))*,  
+IEEE (2025). [PDF](https://ieeexplore.ieee.org/stamp/stamp.jsp?tp=&arnumber=10681084)
```

Comment:
the PDF part is used for those papers or documents that have a PDF available not behind a paywall. 
In your case the paper is open access, so there is no need for the extra PDF link.